src/map/status.cpp

```diff
@@ -1390,6 +1390,8 @@ void initChangeTables(void)
 	StatusIconChangeTable[SC_ENTRY_QUEUE_APPLY_DELAY] = EFST_ENTRY_QUEUE_APPLY_DELAY;
 	StatusIconChangeTable[SC_ENTRY_QUEUE_NOTIFY_ADMISSION_TIME_OUT] = EFST_ENTRY_QUEUE_NOTIFY_ADMISSION_TIME_OUT;
 
+	StatusIconChangeTable[SC_MADOGEAR] = EFST_MADOGEAR;
```

Comment:
Maybe make this depending on packetversion instead and always start the SC.
We seem to have sent out EFST_RIDING before that.